src/visitpy/examples/wave.py

```diff
@@ -22,7 +22,7 @@
 # Check the version.
 ScriptVersion = "1.2.1"
 if(Version() != ScriptVersion):
```

Comment:
There seems to be a ton of code duplicated across these example `.py` files. Maybe add a ticket to refactor?

Reply:
I added #4192